lms/templates/courseware/courses.html

```diff
@@ -45,15 +57,36 @@
   </header>
 
   <section class="courses-container">
+
+    <div id="discovery-form">
+      <form>
+        <input class="discovery-input" placeholder="${_('Search for a course')}" type="text"/><!-- removes spacing
+        --><button type="submit" class="button postfix discovery-submit" aria-label="${_('Search')}">
```

Comment:
The input and button elements are styled as inline elements. To have them adjacent, there must be no whitespace in between.